index.html

```diff
@@ -26,14 +26,15 @@
 <body>
   <nav class="destop-version">
       <a href="#" class="my-logo">Alexander</a>
-    <nav class="hamburger-menu">
+    <div class="open-menu">
```

Comment:
- [ ] With this on your code, the hamburger will be display since the first reload of the page, [open this link](https://rawcdn.githack.com/alexander16108/destop-version/452ecf86e0bcbd32365cb536d43bd6f38d7bf2c8/index.html#) on a mobile phone or developers tool. The expected behavior is when the user click (or tap) the hamburger button, the mobile menu appears.